Source/interfac.cpp

```diff
@@ -216,16 +218,23 @@ void interface_msg_pump()
 	}
 }
 
-bool IncProgress()
+void IncProgress()
 {
 	interface_msg_pump();
 	if (!IsProgress)
-		return false;
+		return;
 	sgdwProgress += 23;
-	if (sgdwProgress > 534)
-		sgdwProgress = 534;
+	if (sgdwProgress > MaxProgress)
+		sgdwProgress = MaxProgress;
 	DrawCutsceneForeground();
-	return sgdwProgress >= 534;
+}
+
+void CompleteProgress()
+{
+	if (!IsProgress)
+		return;
+	while (sgdwProgress < MaxProgress)
+		IncProgress();

```

Comment:
Bummer I had previously managed to align the number of steps so the progressbar would be at the same stage at the same position for both code paths. I guess we got rid of some useless stuff since then :)